src/test/java/com/commercetools/sync/internals/utils/UnorderedCollectionSyncUtilsTest.java

```diff
@@ -0,0 +1,94 @@
+package com.commercetools.sync.internals.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.products.Product;
+import io.sphere.sdk.products.ProductVariant;
+import io.sphere.sdk.products.ProductVariantDraft;
+import io.sphere.sdk.products.ProductVariantDraftBuilder;
+import io.sphere.sdk.products.commands.updateactions.RemoveVariant;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.commercetools.sync.internals.utils.UnorderedCollectionSyncUtils.buildRemoveUpdateActions;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class UnorderedCollectionSyncUtilsTest {
+
+    @Test
+    public void buildRemoveUpdateActions_withNullNewDraftsAndEmptyOldCollection_ShouldNotBuildActions() {
```

Comment:
I just wonder - can you do parameterized test here as well?

Reply:
yes you definitely can, but I though it's more clear this way have a separate method for each case since there are not a lot of different cases.